prestoadmin/package.py

```diff
@@ -18,21 +18,23 @@
 import logging
 
 from fabric.context_managers import settings, hide, shell_env
-from fabric.decorators import task
+from fabric.decorators import task, runs_once
 from fabric.operations import sudo, put, os, local
 from fabric.state import env
+from fabric.tasks import execute
 from fabric.utils import abort
 
 from prestoadmin.util import constants
 from prestoadmin.standalone.config import StandaloneConfig
 from prestoadmin.util.base_config import requires_config
-
+from prestoadmin.util.fabricapi import get_host_list
 
 _LOGGER = logging.getLogger(__name__)
 __all__ = ['install']
 
 
 @task
+@runs_once
 @requires_config(StandaloneConfig)
 def install(local_path):
```

Comment:
Good catch. We need a similar change to server.py's install method to be @runs_once and then execute the remaining steps. See the diff for 82b5cb9b622a94bdc3da9d8ff6429e1c05e545e2 for how it used to be structured.